tools/serve/serve.py

```diff
@@ -40,6 +40,8 @@ class WrapperHandler(object):
 
     __meta__ = abc.ABCMeta
 
+    headers = ()
```

Comment:
A tuple is immutable, a list is mutable (and, argument from purity, tuples are supposed to be things where each field has a predefined meaning, not arbitrary ordered collections).